src/agent/src/main.rs

```diff
@@ -87,6 +89,20 @@ cfg_if! {
 
 const NAME: &str = "kata-agent";
 
+const UNIX_SOCKET_PREFIX: &str = "unix://";
+
+const AA_PATH: &str = "/usr/local/bin/attestation-agent";
+const AA_UNIX_SOCKET_DIR: &str = "/run/confidential-containers/attestation-agent/";
+const AA_ATTESTATION_SOCKET: &str = concatcp!(AA_UNIX_SOCKET_DIR, "attestation-agent.sock");
```

Comment:
It doesn't look like `AA_UNIX_SOCKET_DIR` is called anywhere else, so could we just do:
```
const AA_ATTESTATION_SOCKET: &str = "/run/confidential-containers/attestation-agent/attestation-agent.sock";
```
?

Reply:
1+